module/input/Camera/data/config/Cameras/Left.yaml

```diff
@@ -1 +1,29 @@
+driver: Aravis
+
+buffer_count: 4             # The number of software buffers to use.
+
 is_left: true
+
+format:
+  width: 1280
+  height: 1024
+  pixel: BayerRG8
+
+lens:
+  focal_length: 412.507218876330336              # 1/focal_length_in_pixels = camera_pixel_size / camera_focal_length = 4.8um / 1.98mm
+  centre_offset: [0, 0]
+  fov: 3.14
+
+settings:
+  exposure: 33333             # The shutter time in microseconds (1/fps).
+  gain: 10                  # The gain of the camera in dB.
+  gamma: 1.25                 # For 8-bit gamma is applied as OUT = 255 * (IN/255)^1/gamma. If 0, gamma will be disabled to create a linear response.
```

Comment:
Remove everything but exposure and gain since we are unable to use any of the others

Reply:
Fix it!